statistics/handle/handle.go

```diff
@@ -1991,3 +2052,40 @@ func (h *Handle) CheckHistoricalStatsEnable() (enable bool, err error) {
 	}
 	return variable.TiDBOptOn(val), nil
 }
+
+func (h *Handle) recordHistoricalStatsMeta(tableID int64, version uint64) error {
+	historicalStatsEnabled, err := h.CheckHistoricalStatsEnable()
```

Comment:
Please also check whether `tableID` and `version` are valid: `if tableID==0 || version == 0 ...`

Reply:
> Please also check whether `tableID` and `version` are valid: `if tableID==0 || version == 0 ...`

Done